pi-manage

```diff
@@ -1319,6 +1284,252 @@ def import_tokens(file, tokenrealm=None):
 token_manager.add_command('import', Command(import_tokens))
 
 
+def conf_export(conftype=DEFAULT_CONFTYPE_LIST, filename=None, name=None, print_passwords=False):
+    """
+    Export configurations to a file or write them to stdout if no filename is given.
+    """
+    import pprint
+    pp = pprint.PrettyPrinter(indent=4)
+
+    ret_dict = {}
+    conf = None
+    if isinstance(conftype, list) or isinstance(conftype, tuple):
+        conftype_list = conftype
+    else:
+        conftype_list = [conftype]
+
+    for conftype in conftype_list:
+        if conftype == "policy":
```

Comment:
in order to make this (and `conf_import`) more readable we can get rid of the `if...elif` chains the following way:
- define helper functions like `_export_resolver()` which do the exporting
- in the loop You can do `if '_export_' + conftype in globals(): globals()['_export_' + conftype]()` 
Alternatively we can define a dictionary containing the functions with the appropriate keys.

Reply:
I'll do that for more transparency, although it creates additional lines of code.